layouts/shortcodes/notice.html

```diff
@@ -0,0 +1,27 @@
+{{/* Available notice types: warning, info, update */}}
+{{- $noticeType := .Get 0 -}}
+<div
+  class="notice{{ if eq $noticeType "warning" }}
+    notice--warning
+
+  {{ else if eq $noticeType "update" }}
+    notice--update
+
+  {{ end }}"
+>
+  <div
```

Comment:
As long as the title is just a text label, I'd give it a semantic tag like `<span>`. Being a flex-child, it becomes a block level element. Then just clean it up with some padding.